code/cavity2d/CAVITY_2D_STAGGERED.F90

```diff
@@ -9,499 +9,556 @@
 !============OKOKOK============100.3.12================OKOKOK=================
 !!!!!!!! PLOT:  UY, VX
 !=============================================================================
-MODULE PISO
-parameter(MI=512,MJ=512)
-REAL XL,DT,DTO,DX,DXO,DY,DYO,DZ,DZO,CFL
-INTEGER NX,NXP,NXPP,NXM,NY,NYP,NYPP,NYM,NXH
-REAL GAM,GAMO,GAMM,GAMMO
-REAL DENI,UIN,VIN,WIN,PREI,RMACH,ENGI,RTI,REO
-END MODULE
+      MODULE PISO
```

Comment:
Indentation rules: (i) the starting indent is 6 characters, (ii) each indent level uses 3 characteres, and (iii) no tab is allowed.